### DISCUSSION TITLE
Resize Rectangle

### DISCUSSION THREAD

**Discussion by awolff64** [coreemu/core#719] [Q&A] [answered]
Am painting a complex simulation setup an want to use the recangle feature of annotations.
I can move them around change colors etc.

What I did not manage to is changing its size.
Tried all I can imagine (and searched discussion and the web) but no success.

Can anybody help?

Regards,
Andreas

**Comment by bharnden** [comment, marked as answer]
For now there is no plan to support that, appreciate the idea though. For now you would just delete and recreate to "resize".
